### DISCUSSION TITLE
synergy with "advanced data analytics for multiple microscopies" DOE SBIR FY23 topic

### DISCUSSION THREAD

**Discussion by dwinston** [marda-alliance/metadata_extractors#12] [Ideas]
I am considering submitting a US DOE SBIR Phase I proposal on "advanced data analytics for multiple microscopies: graphical analyses for materials characterization" (on pp21-22 of this PDF: https://science.osti.gov/-/media/sbir/pdf/TechnicalTopics/FY23-Phase-I-Release-2-Combined-TopicsV312072022.pdf).

Some of my notes from the DOE webinar session for the topic (from https://youtu.be/efqsuZUAO9g?t=2456 (40:56) until https://youtu.be/efqsuZUAO9g?t=3547 (59:07)):
* issue: information coming off instruments is all in different formats and different file types
* issue: being able to pull all that information and exploit it on one object (material sample) is challenging in itself; would like to be able to do population comparisons.
* issue: instrumentation manufacturers such as Thermo Industries and FEI may or may not export into open imaging formats and may or may not serve raw data vs processed data and may or may not make on-board processing software source-available/open.
* issue: when doing SEM and you do a large field of view, and then you go through with another technique and you do a much smaller field of view -- e.g. SEM image is a brick, and another technique is imaging a grain of sand on that brick -- we'd like to overlay that grain of sand on the brick, to register those two images. how would you do that registration process between two techniques using very different physical measurement processes?
* position: partner with someone who does imaging using multiple techniques

I propose to perform a content audit and inventory across relevant instrument data output file types and formats, for both raw data and processed data (for instruments that do pre-export processing) as appropriate, in order to identify a domain of concepts and corresponding metadata elements. Prior to this, I plan to survey researchers that have done or plan to do statistical probes of material samples for the purpose of population comparisons across objects using at least three imaging techniques such as SEM, Raman, XRD, XRF, etc., in order to determine a domain of instrument makes/models for the aforementioned content audit and inventory. After identifying a spanning set of metadata concepts, I plan to use the Resource Description Framework (RDF) suite of standards for interoperable data exchange and semantic knowledge representation to model data schema, to map this schema to existing schema as appropriate, and ultimately ensure a programmatic basis for instrument data extraction to a common intermediate format as a unifying first step in various Extract-Transform-Load (ETL) data analysis activities.

I personally have extensive experimental research experience at the graduate student and post-doctoral levels dealing with the nuances of advances imaging and metrology techniques for materials and process characterization, particularly with regard to scanning changed-particle-beam instrumentation such as SEM and FIB. I also have years of experience in my current focus of software and data systems design and engineering, particularly regarding data modeling / schema design, data processing, and metadata management for data-intensive DOE-funded materials science and environmental science projects.

When I saw this topic call, I immediately thought of the MaRDA Extractors WG. What do folks think of this? I plan to submit a letter of intent (LOI) for this soon (deadline is Jan 3). The full application is due Feb 23. Please let me know

1. if you also see potential synergy between this WG's goals and the goals indicated by the DOE topic manager (in the linked PDF above / in the linked webinar above);
2. if you are potentially interested in writing a letter of support (encouraged as attachments to the proposal); and
3. if you recommend anyone that I might get in touch with to partner with for data sourcing, i.e. someone who does / plans to do material sample characterization using multiple imaging/analytical instruments.

**Comment by davidelbert** [comment]
Hi Donny-  
It's an interesting topic on many levels. Certainly extracting metadata from microscopy files is in scope in this group.  It's something many people do.   I'll interject three things just in case you aren't aware of them:  1) there have been several groups with workshops on microscopy metadata/data in the last few years. Efforts from Lehigh, NIST, and CHiMaD.  And just 10 days ago the Materials Research Coordination Network (within MaRDA) ran a virtual meeting to kickstart more work on microscopy data.  That is working towards starting a couple of MaRDA Working Groups.  I can connect you with that effort if you'd like.  and 2) there is some excellent software to translate file formats for TEM microscopy as part of the hyperspy package.  Many other efforts as well, but hyperspy gets leveraged by most.  It covers a lot of formats and has an implementation of HDF5 to call its own.  If you have a microscopy background you probably know it, but just in case not here's a starting point on file formats: https://hyperspy.org/hyperspy-doc/current/user_guide/io.html#supported-formats.  and 3) correlative microscopy is certainly a complicated topic but growing and even includes a commercialized solution known as nanoGPS from Horiba.
-David
